lib/AFS/CellCC.pm

```diff
@@ -50,7 +50,7 @@ _exclude_volume($$$$$) {
     }
 
     my $operation = 'sync';
-    if ($opts->{delete}) {
+    if ($opts->{delete} || $opts->{forcedelete}) {
```

Comment:
(from above)... this code change. We still set CELLCC_FILTER_OPERATION to 'delete', whether we are deleting or force-deleting